velox/functions/prestosql/tests/DateTimeFunctionsTest.cpp

```diff
@@ -485,6 +485,128 @@ TEST_F(DateTimeFunctionsTest, yearTimestampWithTimezone) {
           "year(c0)", std::nullopt, std::nullopt));
 }
 
+TEST_F(DateTimeFunctionsTest, week) {
+  const auto week = [&](StringView date) {
+    return evaluateOnce<int64_t>(
+        "week(c0)",
+        std::make_optional(Timestamp(util::fromTimeString(date), 0)));
+  };
+
+  EXPECT_EQ(1, week("00:00:00"));
```

Comment:
I'm a bit lost about how a time string can have a week. I would've expected this to error. Please can you explain the behavior of these tests.